src/quo2/components/record_audio/record_audio/view.cljs

```diff
@@ -211,13 +211,43 @@
            (fn []
              (audio/destroy-recorder @recorder-ref)
              (reset! recorder-ref nil))
+           recorder-on-meter
+           (fn []
+             (let [current-timestamp  (.now js/Date)
+                   recording-duration (- current-timestamp
+                                         @start-recording-timestamp)]
```

Comment:
Maybe to be extra precise, we should name these bindings with the millisecond unit. It's usually a good practice, as you did with some other bindings.

- `current-timestamp` -> `current-timestamp-ms` or `now-ms`
- `start-recording-timestamp` -> `start-recording-timestamp-ms` or `recording-start-ms` (noun first, since it's not a function, and it pairs well with another binding you named `recording-length-ms`).